src/components/dev-hub/podcast-player/player-listen-on.js

```diff
@@ -0,0 +1,70 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+
+import Button from '~components/dev-hub/button';
+import {
+    AppleLogo,
+    GoogleLogo,
+    SpotifyLogo,
+} from '~components/dev-hub/icons/audio-player';
+
+import { colorMap, size } from '~components/dev-hub/theme';
+
+export const LogosContainer = styled('div')`
+    align-items: center;
+    display: flex;
+    justify-content: center;
+`;
+
+const StyledButton = styled(Button)`
+    margin-right: 30px;
```

Comment:
```suggestion
    margin-right: ${size.large};
```

When close, let's use the `size` values from the theme please.